safe-global/safe-wallet-web
# feat: Move Redefine into new Tx Flow

## What it solves

Part of #2025 

## How this PR fixes it

- Displays Redefine messages in the widget area
- Moves Tenderly into the new folder structure
- Deletes old components

## How to test it

1. Open a Safe on Staging Mainnet or Polygon
2. Create transactions
3. Observe the Redefine module running

## ToDos

- [x] Adjust message design according to Figma
- [x] Reuse the same styles for Redefine and Tenderly
- [x] Resolve TODOs (remove old components, adjust folder structure but likely after removing old modals)

## Screenshots
<img width="1148" alt="Screenshot 2023-06-26 at 13 51 36" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/c8fa2393-2d12-4bb4-9ce0-5d56a9e6e1d6">


## Checklist
